pkgs/tools/wayland/sirula/default.nix

```
@@ -8,18 +9,27 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sirula";
-  version = "1.0.0";
+  version = "unstable-2023-09-02";
 
   src = fetchFromGitHub {
     owner = "DorianRudolph";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-C5mVO10+jD4TDg6R9rVJO6fdDiOD5tT+bEaI53WVdFA=";
+    rev = "b15efe85ef1fe50849a33e5919d53d05f4f66090";
+    sha256 = "sha256-S0WbqY49nKaBUMWfgDKZxFLJuk7uFcnTfV8s86V0Zxs=";
```

Since we're version bumping anyway

```suggestion
    hash = "sha256-S0WbqY49nKaBUMWfgDKZxFLJuk7uFcnTfV8s86V0Zxs=";
```